packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -781,7 +823,7 @@
 			"#": "87",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "LED - temperature for red colour",
-			"description": "Minimum temperature resulting in red LED illumination",
+			"description": "When the temperature is above this (°C), the light is blue.",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.